ephios/settings.py

```diff
@@ -32,7 +32,7 @@
 DEBUG = env.bool("DEBUG")
 ALLOWED_HOSTS = env.list("ALLOWED_HOSTS")
 
-EPHIOS_VERSION = "v" + importlib_metadata.version("ephios")
+EPHIOS_VERSION = importlib_metadata.version("ephios")
```

Comment:
Yes. I thought about adding it in the footer template but I think it's also fine without isn't it?

Reply:
i think i like it shlighty more with the v, but no hard feelings

Reply:
You know I was thinking reading the footer flows better with the v there. Otherwise your mind has to add the "version" when reading. I'll add it back in...